fedimint-core/src/module/mod.rs

```diff
@@ -809,6 +840,27 @@ pub trait ServerModule: Debug + Sized {
     }
 
     /// This module's contribution to the next consensus proposal
+    ///
+    /// This method is polled by the server periodically to gather
+    /// consensus items modules would like included in the consensus.
+    ///
+    /// Note that returning an item does not guarantee its processing,
+    /// and modules should keep proposing the difference between
+    /// the desired state and current consensus view.
+    ///
+    /// Warning: The consensus algorithm will deduplicate and ignore
+    /// previously submitted items. This can lead to weird behavior
+    /// when identically deserialized items can be generated multiple times.
+    /// Use some random-like data to make such requests unique to avoid
+    /// this problem.
```

Comment:
Is this still the case? I think you mentioned on one of our calls that this was fixed @joschisan?

Reply:
Oh, really? That would be important to know. Please link to a PR if that's the case. I'm curious how it is done.